README.md

```diff
@@ -1,10 +1,48 @@
+See: [LICENSE file](https://github.com/ValkyrienWarfare/Valkyrien-Warfare-Revamped/blob/8778e9d45f16c1f60e8149ab6cbdbabbdebc1278/LICENSE)
+
+The Airships Mod to end all other Airships Mods. Better compatibility, performance, collisions, interactions and physics than anything prior!
+
+
+## Installation
+
+### Downloading
+Official and stable releases of the Valkyrien Warfare mod can be found on the [CurseForge page](https://minecraft.curseforge.com/projects/valkyrien-warfare/files).
+
+Beta releases (warning: may be unstable!) can be found on the [Jenkins build server](https://jenkins.daporkchop.net/job/Valkyrien-Warfare-Revamped/).
+
+### Installing on your server
+To install Valkyrien Warfare, move the downloaded `.jar` file into your Minecraft's `mods/` folder, just as you would any other mod.
+
+## Usage
+
+### Make a flying ship
+1. Build a structure somewhere in the air, not connected to the ground.
+2. Place a [**Physics Infuser**] on it.
+3. Right click the [**Physics Infuser**] (the ship should fall).
+4. Place a [**Pilot Chair**] on the ship.
+5. Right click the [**Pilot Chair**] to get in, and then use <kbd>W</kbd>, <kbd>A</kbd>, <kbd>S</kbd>, <kbd>D</kbd>, <kbd>X</kbd>, and <kbd>Space</kbd> to navigate.
+
+### Make your ship hover without a pilot
+
+1. Place (multiple) [**Ether Compressor**] on your ship
+2. Place a [**Hovercraft Controller**] on your ship.
+3. Right click the [**Hovercraft Controller**] with [**System Linker**].
+4. Right click all [**Ether Compressor**] with [**System Linker**].
+5. Right click the [**Hovercraft Controller**] with your hand and set the **Hover Height Target** at the Y-level you wish for your ship to hover at.
 
-## The Airships Mod to end all other Airships Mods. Better compatibility, performance, collisions, interactions and physics than anything prior!
+## Development
 
-## Also the airship mod of The Future®
+### Eclipse
+1. Clone the repo: `git clone https://github.com/ValkyrienWarfare/Valkyrien-Warfare-Revamped`
+2. Copy in the `eclipse/` folder from a fresh installation of the [Forge MDK](http://files.minecraftforge.net)
+3. Run `./gradlew setupDecompWorkspace eclipse`
+4. Open the project in Eclipse
 
-### Setting up
-Setting this mod up is exactly the same as with any other mod, just throw it in your `mods` folder and start MC! If you want to develop for it, clone the repo, (if you're using Eclipse you'll also need to copy in the `eclipse/` folder from a fresh installation of the [Forge MDK](http://files.minecraftforge.net), ) then use `./gradlew setupDecompWorkspace eclipse` (Linux/Mac + Eclipse) or `./gradlew setupDecompWorkspace genIntellijRuns` (Linux/Mac + IntelliJ) or `gradlew.bat setupDecompWorkspace eclipse` (Windows + Eclipse) or `gradlew.bat setupDecompWorkspace genIntellijRuns` (Windows + IntelliJ).
+### IntelliJ
+1.  Clone the repo: `git clone https://github.com/ValkyrienWarfare/Valkyrien-Warfare-Revamped`
+2. Run `./gradlew setupDecompWorkspace idea genIntellijRuns`
```

Comment:
won't work in windows last i checked (i could also just not have used it for a long time), iirc you have to use `gradle.bat`

Reply:
Works w/ powershell, anyone contributing to this will likely know how to execute a batch script - not a big concern

Reply:
didn't realize that powershell was the new standard, my bad